File: RedstoneLib/Types/Difficulty.cs
namespace Redstone.Types
{
    /// <summary>
    /// Difficulty setting of Minecraft
    /// </summary>
    public enum Difficulty : byte
    {
        /// <summary>
        /// The easiest of all the difficulties.
        /// Hunger does not deplete, and there are no monsters.
        /// Death can still happen by natural causes.
        /// Health regenerates.
        /// </summary>
        Peaceful = 0,

        /// <summary>
        /// Hunger slowly depletes, and monsters are easiest.
        /// Health does not regenerate.
        /// </summary>
        Easy = 1,

        /// <summary>
        /// Slightly more difficult, plus monsters have armor.
        /// </summary>
        Normal = 2,

        /// <summary>
        /// The hardest. You will not survive.
        /// </summary>
        Hard = 3,
    }
}

File: src/com/CopperPenguin96/Redstone/System/Types/Difficulty.java
package com.CopperPenguin96.Redstone.System.Types;

public enum Difficulty {
	Peaceful(0),
	Easy(1),
	Normal(2),
	Hard(3);
	
	public int _val;
	private Difficulty(int val) {
		_val = val;
	}
}
